### PR TITLE
Update CI status badge to reflect correct repo

### PR DESCRIPTION
**Pull request by ExtraShotLatte**
Updated CI status badge to correctly point to our repo's.

Contributes to #23

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![CI Status](https://github.com/AY2223S2-CS2103T-T11-3/tp/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
```
Comment (bryansendeavour) [inline README.md:1]: Maybe the path of the link at the back should also be changed to AY2223S2-CS2103T-T11-3/tp instead of se-edu/addressbook-level3 ?

**Review by bryansendeavour — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ExtraShotLatte) [reply README.md:1]: MB, didn't notice this! Will update

Comment (ExtraShotLatte) [reply README.md]: Done!

**Review by bryansendeavour — APPROVED** [review]
LGTM

**Pull request merged**